simulator/channel.h

```diff
@@ -13,6 +13,10 @@
 #include <numeric>
 
 #include "armadillo"
+#include "channel_models/awgn_model.h"      //Testing
```

Comment:
you can add this file path to the cmake.  channel_models  .  Remove the testing comments